airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/order.json

```diff
@@ -297,17 +285,21 @@
               "partially_delivered",
               "returned",
               "cancelled"
-            ],
-            "items": {
-              "type": "string"
-            }
+            ]
           },
           "entity_type": {
-            "type": "array",
-            "enum": ["adhoc"],
-            "items": {
-              "type": "string"
-            }
+            "type": ["string", "null"],
+            "enum": [
+              "adhoc",
+
```

Comment:
Why so?

Reply:
Did you mean why leave empty lines? If yes then since Chargebee has 2 schema versions (Product Catalog 1.0 and Product Catalog 2.0) for some fields values can also have 2 versions. Empty lines just for visual separation:
```
common

fields for 1.0

fields for 2.0
```